### PR TITLE
New Uberonparcellation terms

### PR DESCRIPTION
**Pull request by spieschnik**
#318

### CODE REVIEW

File: instances/UBERONParcellation/entorhinalCortexLayer5.jsonld [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/entorhinalCortexLayer5",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
+    "definition": "'Entorhinal cortex layer 5' is a layer of the entorhinal cortex.",
+    "description": "Layer of entorhinal cortex lying superficial to layer 6 and deep to layer 4 characterized by large pyramidal neurons that are darkly stained in Nissl preparations (Adapted from Paxinos, G. The rat central nervous system, 2nd ed, Academic Press, San Diego, 1995, pg. 474)",
```
Comment (UlrikeS91) [inline instances/UBERONParcellation/entorhinalCortexLayer5.jsonld]: Dot missing at the end of the sentence.

File: instances/UBERONParcellation/entorhinalCortexLayer5.jsonld [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/entorhinalCortexLayer5",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
+    "definition": "'Entorhinal cortex layer 5' is a layer of the entorhinal cortex.",
```
Comment (UlrikeS91) [inline instances/UBERONParcellation/entorhinalCortexLayer5.jsonld]: 'Entorhinal cortex layer 5' is one of the deep layer of the entorhinal cortex.

File: instances/UBERONParcellation/entorhinalCortexLayer6.jsonld [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/entorhinalCortexLayer6",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
+    "definition": "'Entorhinal cortex layer 6' is the latter layer of the entorhinal cortex.",
```
Comment (UlrikeS91) [inline instances/UBERONParcellation/entorhinalCortexLayer6.jsonld]: 'Entorhinal cortex layer 6' is the innermost layer of the entorhinal cortex.

File: instances/UBERONParcellation/entorhinalCortexLayer6.jsonld [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    "description": "Layer of entorhinal cortex lying superficial to the cerebral white matter and deep to layer 5 characterized by cells of various shapes and sizes (Adapted from Paxinos, G. The rat central nervous system, 2nd ed, Academic Press, San Diego, 1995, pg. 474).",
+    "interlexIdentifier": "http://uri.interlex.org/base/ilx_0103867",
+    "knowledgeSpaceLink": "https://knowledge-space.org/wiki/NLX:98#entorhinal-cortex-layer-6",
```
Comment (UlrikeS91) [inline instances/UBERONParcellation/entorhinalCortexLayer6.jsonld]: https://knowledge-space.org/wiki/UBERON:0022329#entorhinal-cortex-layer-6

File: instances/UBERONParcellation/entorhinalCortexLayer5.jsonld [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    "description": "Layer of entorhinal cortex lying superficial to layer 6 and deep to layer 4 characterized by large pyramidal neurons that are darkly stained in Nissl preparations (Adapted from Paxinos, G. The rat central nervous system, 2nd ed, Academic Press, San Diego, 1995, pg. 474)",
+    "interlexIdentifier": "http://uri.interlex.org/base/ilx_0103866",
+    "knowledgeSpaceLink": "https://knowledge-space.org/wiki/NLX:58#entorhinal-cortex-layer-5",
```
Comment (UlrikeS91) [inline instances/UBERONParcellation/entorhinalCortexLayer5.jsonld]: https://knowledge-space.org/wiki/UBERON:0022325#entorhinal-cortex-layer-5

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

File: instances/UBERONParcellation/entorhinalCortexLayer5.jsonld (hunk 2 above)
Comment (spieschnik) [reply instances/UBERONParcellation/entorhinalCortexLayer5.jsonld]: 'Entorhinal cortex layer 5' is one of the deep layers of the entorhinal cortex.

**Review by UlrikeS91 — APPROVED** [review]

**Pull request merged**